src/Servers/IIS/IIS/test/Common.FunctionalTests/Infrastructure/EventLogHelpers.cs

```diff
@@ -161,18 +161,18 @@ public static string InProcessStarted(IISDeploymentResult deploymentResult)
 
     public static string OutOfProcessStarted(IISDeploymentResult deploymentResult)
     {
-        return $"Application '/LM/W3SVC/1/ROOT' started process '\\d+' successfully and process '\\d+' is listening on port '\\d+'.";
+        return $"Application '/LM/W3SVC/\\d+/ROOT' started process '\\d+' successfully and process '\\d+' is listening on port '\\d+'.";
```

Comment:
Why are we loosening from strictly matching on `1` to any digit here?

Reply:
Multi-site handling was added to add test coverage for the regression. Technically we might be able to pass in the site id to these methods so it's not using regex.